src/classes/services/ContactService.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactService {
+    public static void createTeamMember(List<Contact> newContacts){
+
+        List<Contact> contactList = new List<Contact>();
+
+        List<Boutique_Resource__c> boutiqueList = new List<Boutique_Resource__c>();
+
+        for (Contact item: newContacts){
+            if (item.AccountId != null) {
+                contactList.add(item);
+            }
+        }
+
+        for (Contact item: contactList){
+            Boutique_Resource__c newTeamMember = new Boutique_Resource__c(Boutique__c = item.AccountId, Contact__c = item.Id, Name = item.LastName);
+            newTeamMember.RecordTypeId = '0127S000000c4WtQAI';
+            boutiqueList.add(newTeamMember);
+        }
+
+        insert boutiqueList;
+
+    }
+
+    public static void updateTeamMember(List<Contact> newContacts, Map<Id,Contact> oldMapContacts, Map<Id,Contact> newMapContacts){
```

Comment:
MINOR
Naming issues are the same as for the method above.
Also, it is enough just to pass new and old Map, list is just not needed.

Reply:
insted of List<Contact> newContacts --> Trigger.new ?

Reply:
ohh, i got it. just pass 2 maps